File: listJobs.pl
#!C:\xampp\perl\bin\perl.exe
print "Content-type: text/html\n\n";
use CGI;
use CGI::Carp qw(fatalsToBrowser);
$cgi=new CGI;
read(STDIN,$FormData,$ENV{'CONTENT-LENGTH'});

$qualification=$cgi->param('qualification');

use DBI;

$DSN="driver=Microsoft Access Driver (*.mdb, *.accdb);dbq=C:\\Users\\Seeni\\Documents\\JobKar.accdb";

my $dbh = DBI->connect("dbi:ODBC:$DSN") or 
  die "$DBI::errstr\n";

print("Connection Sucessful<br />");

$sth=$dbh->prepare("Select * from Jobs;");

$sth->execute();

while(my @row  = $sth->fetchrow_array()){
  ($qual,$location,$noa)=@row;
  if($qual eq $qualification){
    print "@row[0] @row[1] @row[2] <br />";
    if($noa > 0){
      $noa-=1;
      $q="Update Jobs set noa=? where qual=? and location=?;";
      $upq=$dbh->prepare($q);
      $upq->execute($noa,$qual,$location) or die "$DBI::errstr<br />"; 
    }
    else{
      $q="Delete from Jobs  where qual=? and location=? and noa=? ;";
      print($q);
      $dq=$dbh->prepare($q);
      $dq->execute($qual,$location,$noa) or die "$DBI::errstr<br />";   
    }
  }
}

File: listJobs.php
<?php

$servername="localhost:3306";
$username="root";
$password="";
$qualification=$_GET["qualification"];

echo "qualification:".$qualification."<br/>";

//conncetion
$conn= mysqli_connect($servername,$username,$password,"jobkar");

if(!$conn){
	print ("Connection Error");
}
echo "connected sucessfully<br/>";

#select all matching jobs and noa >0
$sq="SELECT * FROM JOBS where qual=\"$qualification\" and noa > 0";
$result=mysqli_query($conn,$sq);
if(mysqli_num_rows($result)>0){
	echo "QUALIFICATION   LOCATION   NOA<br/>";
	while ($row=mysqli_fetch_array($result) ){
				$qual=$row['qual'];
				$location=$row['location'];
				$noa=(int)$row['noa']-1;
				print("{$qual} {$location} {$noa} <br />");
				#subtract the count and update it
				$uq="UPDATE jobs set noa = {$noa} where qual = \"{$qualification}\" and location =\"{$location}\"";
				mysqli_query($conn,$uq);
	}
}

#delete records whose noa is 0
$sq="DELETE FROM JOBS where noa = 0";
mysqli_query($conn,$sq);

mysqli_close($conn);
?>